organizations/models.py

```diff
@@ -31,6 +33,9 @@ class Organization(TimeStampedModel):
     )
     active = models.BooleanField(default=True)
 
+    def __str__(self):
+        return u"Organization object"
```

Comment:
Adding this method with the new class decorator removes the need for `__unicode__()`; do you want to just move that logic here?

Reply:
Good idea - done.